style.css

```diff
@@ -46,64 +51,148 @@ body {
   border-radius: 50%;
 }
 
-/* Меню */
 .menu {
   display: flex;
   align-items: center;
 }
 
 .menu__item {
-  padding: 0 13px;
+  position: relative;
+  margin-right: 26px;
   text-decoration: none;
   font-weight: bold;
   font-style: normal;
   font-size: 18px;
   line-height: 21px;
   color: #000;
+  transition: opacity .5s;
 }
 
 .menu__item:last-child {
-  padding-right: 0;
+  margin-right: 0;
 }
 
-.main-section {
-  display: flex;
+.menu__item:after {
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+  opacity: 0;
+  transition: opacity .5s;
+}
+
+.menu__item:hover {
+  opacity: .5;
+}
+
+.menu__item:hover:after,
+.menu__item_active:after {
+  opacity: 1;
+}
+
+.burger{
+  display: none;
 }
 
-.main-content {
-  margin-top: 105px;
+.main {
+  padding: 0 63% 100px 0;
+  background: url(images/mainimg.svg);
+  background-repeat: no-repeat;
+  background-position: top right;
 }
 
-.main-content__title {
+.main__title {
   font-family: PT Serif, Helvetica, Arial, Verdana, Tahoma, sans-serif;
   font-style: normal;
   font-weight: bold;
-  font-size: 66px;
-  line-height: 87px;
+  font-size: 50px;
+  line-height: 65px;
   align-items: center;
   color: #000;
+  margin: 0 0 40px 0;
 }
 
-.main-content__subtitle {
+.main__title-text_highlight {
+  background: #ecf3ff;
+}
+
+.main__subtitle {
   font-style: normal;
   font-weight: normal;
-  font-size: 22px;
-  line-height: 26px;
+  font-size: 18px;
+  line-height: 21px;
   color: #000;
+  margin: 0 0 41px 0;
 }
 
-.main-content__button {
+.main__button {
+  display: inline-block;
   font-style: normal;
   font-weight: 500;
-  font-size: 18px;
-  line-height: 21px;
+  font-size: 14px;
+  line-height: 16px;
   color: #fff;
   background: #0056ec;
   border-radius: 4px;
-  padding: 10px 21px;
+  padding: 14px 16px;
+  text-decoration: none;
+  transition: opacity .5s;
 }
 
-.image-content__image {
-  width: 770px;
-  height: 657px;
+.main__button:hover {
+  opacity: .5s;
+  background: red;
 }
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .header {
+    margin-bottom: 28px;
+  }
+
+  .logo__shadow {
+	display: none;
+  }
+
+  .burger {
+    display: block;
+    width: 25px;
+    height: 17px;
+    padding-top: 33px;
+  }
+
+  .main {
+    padding: 74% 0 0;
+    background-size: 74%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin-bottom: 44px;
+    font-family: PT Serif, Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-size: 30px;
+    line-height: 40px;
+    text-align: center;
+  }
+
+  .main__subtitle {
+    margin-bottom: 25px;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)